Configuration/commands.json

```diff
@@ -245,6 +245,16 @@
             },
             "category": "Fun 🎪"
         },
+        "dogfact": {
+            "usage": "",
+            "description": "Random fact about dogs",
```

Comment:
The command allows for multiple facts — maybe this should be "Random fact(s) about dogs" instead?